src/vercel/server.ts

```diff
@@ -0,0 +1,91 @@
+// import { createCache } from "./deps.ts";
+import assets from "./../assets.ts";
+import transform from "./../transform.ts";
+import render from "./_render.ts";
+// import preloader, { ultraloader } from "./../preloader.ts";
+import { jsxify, tsxify } from "./../resolver.ts";
+// import { isDev } from "./../env.ts";
+
+import { StartOptions, VercelStart } from "./../types.ts";
+
+const server = (
+  { importmap, dir = "src", root = "http://localhost:8000", lang = "en" }:
+    StartOptions,
+) => {
+  // const cache = createCache({ root: "/tmp" });
+  // const fileRootUri = `file://${Deno.cwd()}/${dir}`;
+  // const link = await ultraloader({ importmap, cache });
+  // const serverStart = +new Date();
+
+  const handler = async ({ request }: VercelStart) => {
+    // const requestStart = +new Date();
+    const cacheBuster = -1;
+    const { transpile } = await assets(dir);
+    const url = new URL(request.url);
+
+    const transpilation = async (file: string) => {
+      const headers = {
+        "content-type": "application/javascript",
+      };
+
+      const source = await Deno.readTextFile(`./${file}`);
+      const t0 = performance.now();
+      const js = await transform({
+        source,
+        importmap,
+        root,
+        cacheBuster,
+      });
+      const t1 = performance.now();
+      console.log(`Transpile ${file.replace(dir, "")} in ${t1 - t0}ms`);
+
+      // const link = await preloader(
```

Comment:
Figure this out at a later time?

Reply:
Yea, Vercel is a bit more work to get working to the level of Deno Deploy. Mainly, not having a Deno compatible way to build on Vercel (yet). https://github.com/vercel-community/deno/issues/95

I wanted to keep this in, because it theoretically does work, just not at the quality or speed needed yet. Maybe I will remove this and put it on its own WIP branch.